docs/source/models/gbd2017/causes/neural_tube_defects/index.rst

```diff
@@ -68,6 +68,32 @@ separately and then fit to a total model of all neural tube defects.
    [GBD-2017-YLD-Capstone-Appendix-1-Neural-Tube-Defects]_ and from the `CoD Capstone
    <http://dx.doi.org/10.1016/S0140-6736(18)32203-7>`_ Appendix.
 
+Cause Hierarchy
++++++++++++++++
+
+cause id: 642
+
+level: 4
+
+Sequelae and health states associated with Neural Tube Defects
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The Neural Tube Defects cause has 85 sequelae, which can be summarized as follows:
+
+- All infants with anencephaly are assigned the health state of severe motor and
+  cognitive impairment.
+- Cases of spina bifida and encephalocele are split into
+  every combination of mild, moderate and severe motor impairment, all severities
```

Comment:
how many ID severities are there?  I proposing including here as "all X severities of ID."

Reply:
It's probably the same as for motor impairment (mild, moderate, severe), unless there's also an 'asymptomatic' category.  I'll check and incorporate this in the next pull request.